code/go/0chain.net/smartcontract/vestingsc/config.go

```diff
@@ -254,15 +273,10 @@ func (vsc *VestingSmartContract) getConfigHandler(
 	params url.Values,
 	balances chainstate.StateContextI,
 ) (interface{}, error) {
-	res, err := vsc.getConfig(balances)
+	conf, err := vsc.getConfig(balances, true)
```

Comment:
The getConfig(balances, true) will save local config to MPT if it does not present, should we update the MPT for this `get` request.

Reply:
this will update mpt, as that would only be done if the config is not yet present in mpt. just to ensure config resides in mpt.